index.html

```diff
@@ -1083,6 +1083,30 @@ <h2>Help your friends find some good learning resources</h2>
                 </div>
             </div>
             <!-- Devraj's card end -->
+            <!-- Sumit card -->
+            <div class="card">
+                <p class="name">Sumit Singh</p>
+                <p class="contact">
+                    <i class="fab fa-Github"></i>
+                    <a href="https://github.com/Sumit-14Singh" target="_blank">Sumit Singh</a>
+                </p>
+                <p class="about">Hey I m Sumit Singh, i m new in the field of coding</p>
+                <div class="resources">
+                    <p>3 Useful Web Dev Resources</p>
+                    <ul>
+                        <li>
+                            <a href="https://howhttps.works/" target="_blank" title="First Resource">How https works</a>
+                        </li>
+                        <li>
+                            <a href="https://howdns.works/" target="_blank" title="Second Resource">For DNS Works</a>
```

Comment:
"For DNS Work" is this "How DNS Work"?

Reply:
yes

Reply:
can you fix that typo and also resolve the conflict